code/parking-enterprise/parking-enterprise.css

```diff
@@ -167,3 +167,129 @@ a.small-button {
 /* END BUTTONS */
 
 }
+
+/*********************************************/
+/******** Header Banner Image Styling ********/
+/*********************************************/
+
+/* Base styles for the image */
+img.knHeader__logo-image {
+  width: 100% !important;
+  height: auto !important;
+  max-width: 100% !important;
+  object-fit: contain !important;
+}
+
+/* Base styles for the parent container */
+.knHeader__logo.knHeader__logo--custom {
+  height: auto !important;
+  max-width: 1000px !important;
+  margin: 0 0 0 0 !important; /* Left-justified */
+}
+
+/* Overrides for conflicting styles */
+.knHeader .knHeader__logo--custom {
+  width: 100% !important;
+  max-width: 1000px !important;
+  margin: 0 0 0 0 !important; /* Left-justified */
+}
+
+.knHeader .knHeader__logo .knHeader__logo-image {
+  width: 100% !important;
+  object-fit: contain !important;
+}
+
+/* Media queries for larger screens */
+@media (min-width: 768px) {
+  .knHeader__logo.knHeader__logo--custom {
+    width: 80% !important;
+  }
+}
+
+@media (min-width: 1024px) {
+  .knHeader__logo.knHeader__logo--custom {
+    width: 80% !important; /* Medium width on large screens */
+    max-width: 1000px !important;
+  }
+}
+
+/* Media queries for image resizing */
+@media (max-width: 767px) {
+  .knHeader__logo.knHeader__logo--custom {
+    width: 100% !important; /* Full width on small screens */
+    max-width: 600px !important;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
+  .knHeader__logo.knHeader__logo--custom {
+    width: 80% !important; /* Medium width on medium screens */
+    max-width: 800px !important;
+  }
+}
```

Comment:
This commit has some extra code, please revise. Code file should end at line 229